src/Containers/RideProgressCard/README.md

```diff
@@ -0,0 +1,3 @@
+# Ride Progress Card
+
+This card shows the ride's origin and destination points, and also can indicate the progress of the ride.
```

Comment:
It is better to add a screenshot to illustrate your component. Look at `ProgressBar` component for an example.